tools/Gen2Master/MoveFromGeneration2Master.ps1

```diff
@@ -299,8 +299,9 @@ Function Move-Generation2MasterHybrid {
             dotnet sln $SolutionPath add (Join-Path -Path (Join-Path -Path $DestPath -ChildPath $submoduleDir.Name) -ChildPath Az.$submoduleName.csproj)
 
             # Update psd1
-            $DestPsd1Path = Join-Path -Path (Join-Path -Path $DestPath -ChildPath $ModuleName) -ChildPath "Az.$ModuleName.psd1"
-            $Psd1Metadata = Import-LocalizedData -BaseDirectory (Join-Path -Path $DestPath -ChildPath $ModuleName) -FileName "Az.$ModuleName.psd1"
+            $Psd1FolderPostfix = (Test-Path (Join-Path -Path (Join-Path -Path $DestPath -ChildPath $ModuleName) -ChildPath "Az.$ModuleName.psd1")) ? '' : '.Management'
```

Comment:
Ternary operator is a new feature in PowerShell 7, our pipeline doesn't support this yet.

Reply:
@wyunchi-ms I have replaced it with if-clause.